lib/t/feeds.ex

```diff
@@ -45,9 +45,15 @@ defmodule T.Feeds do
 
     for user_id <- user_ids do
       broadcast(topic, {__MODULE__, :deactivated, user_id})
+      schedule_deactivated_push_notification(user_id)
     end
   end
 
+  defp schedule_deactivated_push_notification(user_id) do
+    job = DispatchJob.new(%{"type" => "deactivated", "user_id" => user_id})
+    Oban.insert(job)
```

Comment:
this would insert each job separately -- 100 users -> 100 inserts. It needs to be done in one insert

Reply:
for example, because you are using the wrong place to schedule the notifications, you can't optimise this insert